docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -1,387 +1,299 @@
-    --enable-debug --enable-sanitizers  --disable-strip --disable-user \
-    --disable-capstone --disable-vnc --disable-sdl --disable-gtk
-cd build && ninja
-```
-
-{{% /tab %}}
-{{% /tabs %}}
-
-Add `export QEMU_ESP32_XTENSA=<path-to-clone-qemu>/build/` to your `.zshrc` or `.bashrc`, or save this command to run in your terminal every session you wish to use the QEMU emulator.
-
-### Build with the QEMU ESP32 Emulator
-
-Navigate to the root of the Micro-RDK repository.
-Once you've `cd`'d to the correct repository, run `. $HOME/esp/esp-idf/export.sh` if you haven't done so already in this terminal session.
-
-You will need to comment out two lines from the file `sdkconfig.defaults`:
-
-``` editorconfig
-CONFIG_ESPTOOLPY_FLASHFREQ_80M=y
-CONFIG_ESPTOOLPY_FLASHMODE_QIO=y
-```
-
-You can then run:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-make sim-local
-```
-
-Or, if you want to connect a debugger:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-make debug-local
-```
-
-### Troubleshooting
-
-If you are unable to connect to the ESP32-backed robot as a remote, try adding `:4545` to the end of the value set in the remote's `Address` field.
-
-You can find additional assistance in the [Troubleshooting section](/appendix/troubleshooting/).
-
-{{< snippet "social.md" >}}
+---
+title: "Microcontroller Setup: the Micro-RDK"
+linkTitle: "Microcontroller Setup"
+weight: 50
+no_list: true
+type: docs
+image: "installation/thumbnails/esp32-espressif.png"
+imageAlt: "E S P 32 - espressif"
+images: ["/installation/thumbnails/esp32-espressif.png"]
+description: "Set up the Espressif ESP32 with the micro-RDK."
+aliases:
+    - /installation/microcontrollers
+# SMEs: Nicolas Menard
+---
+
+{{% readfile "/static/include/micro-rdk.md" %}}
+
+## Hardware Requirements
+
+You need an an Espressif ESP32 microcontroller to use the micro-RDK.
+Viam recommends purchasing the ESP32 with a development board: see [development kit options](https://www.espressif.com/en/products/devkits).
+
+Minimal configuration: 384kB Ram 4MB flash
+Recommended configuration: 384kB Ram + 8MB SPIRAM + 4MB Flash
+
+## Software Requirements
+
+The micro-RDK is written in Rust.
+
+We recommend using Viam's [Canon utility](#using-canon) to build the `micro-rdk` server, as the utility downloads and installs a Docker development environment automatically.
+You can set up the development environment manually by following [these instructions](#set-up-development-environment-manually).
+
+To use the micro-RDK with your ESP32 board, follow these steps:
+
+- [Install prerequisites](#install-prerequisites)
+
+If using the Docker environment with Canon:
+
+- [Use Canon](#using-canon)
+
+If setting up the environment yourself (Optional):
+
+- [Set up development environment manually](#set-up-development-environment-manually)
+
+The following instructions cover installation for macOS and Linux machines.
```

Comment:
as opposed to on the ESP32? I think this could do with a bit more guidance

Reply:
I changed machine to systems, the esp32 cannot host any of this tools but we need them to program the esp32. I have added a sentence in that spirit